src/hooks/mongodb3x.test.js

```diff
@@ -1,31 +1,33 @@
 import { HandlerInputsBuilder } from '../../testUtils/handlerInputsBuilder';
-import {
-  getMockedMongoClient,
-  promisifyMongoFunc,
-  wrapMongoCollection,
-} from '../../testUtils/mongoMocker';
+import { getMockedMongoClient, wrapMongoCollection } from '../../testUtils/mongo3xMocker';
 import { MongoSpanBuilder } from '../../testUtils/mongoSpanBuilder';
 import { SpansContainer, TracerGlobals } from '../globals';
 import { hookMongoDb } from './mongodb';
 
 const DUMMY_URL = 'mongodb://localhost:27017/myproject';
 
 describe('mongodb', () => {
+  let connection;
+
   beforeEach(() => {
     const handlerInputs = new HandlerInputsBuilder().build();
     TracerGlobals.setHandlerInputs(handlerInputs);
   });
 
+  afterEach(() => {
+    connection && connection.close();
+  });
+
   test('hookMongoDb -> simple flow', async () => {
-    const { mongoLib, mongoClient } = getMockedMongoClient();
+    const { mongoClientLibrary, mongoClient } = getMockedMongoClient();
 
-    hookMongoDb(mongoLib);
-    const connection = await promisifyMongoFunc(mongoClient.connect)(DUMMY_URL, {});
+    hookMongoDb(mongoClientLibrary);
+    connection = await mongoClient.connect(DUMMY_URL, {});
     const collection = connection.db().collection('documents');
     wrapMongoCollection(collection, 'insert');
 
     const docs = [{ a: 1 }, { a: 2 }, { a: 3 }];
-    await promisifyMongoFunc(collection.insert)(docs);
+    await collection.insert(docs);
```

Comment:
Why did you add the `promisifyMongoFunc` to the mock? the actual library is not promised as far as I know

Reply:
i guess i got overexcited - the 4.x client uses promises, i'll reset the 3.x client mock to what it was